cmip6_downscaling/tasks/common_tasks.py

```diff
@@ -24,13 +25,48 @@
     make_coarse_obs_path,
     make_interpolated_gcm_path,
     make_interpolated_obs_path,
+    make_pyramid_path,
 )
-from cmip6_downscaling.workflows.utils import rechunk_zarr_array_with_caching, regrid_ds
+from cmip6_downscaling.workflows.utils import BBox, rechunk_zarr_array_with_caching, regrid_ds
 
 get_obs_task = task(get_obs)
 get_gcm_task = task(get_gcm)
 
 
+@task
+def build_bbox(latmin: str, latmax: str, lonmin: str, lonmax: str) -> dataclass:
+    """Build bounding box out of lat/lon inputs using BBox data class defined in /utils.py
+
+    Args:
+        latmin : str
+            Latitude Minimum
+        latmax : str
+            Latitude Maximum
+        lonmin : str
+            Longitude Minimum
+        lonmax : str
+            Longitude Maximum
+
+    Returns:
+        BBox: dataclass for bounding box
+    """
+
+    return BBox(latmin=latmin, latmax=latmax, lonmin=lonmin, lonmax=lonmax)
```

Comment:
I don't understand why this needs to be a task. Is this because we need a task to unpack the Prefect Parameters?

Reply:
If `create_bbox` and `build_time_period_slices` weren't encapsulated in tasks, prefect would complain about unused parameters. I'll add a comment.